readme.md

```diff
@@ -1,5 +1,113 @@
+- When applicable, project must use reproducible build settings in its toolchain.
+- The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
+- The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
+- The project code signs their artifacts as appropriate.
+
+**Licenses and Copyright**
+
+- The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
+- Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
+- Committers are bound by a Contributor License Agreement.
+- The copyright ownership of everything that the project produces is clearly defined and documented.
+
+**Quality**
+
+- The project is open and honest about the quality of its code.
+- The project has a public issue tracker where the status of any defect can be easily obtained.
+- The project puts a very high priority on producing secure software. 
+- The project has a published Security Policy.
+
+**Community**
+
+- The project has a home page which provides high level information about its status and purpose.
+- The project welcomes contributions from anyone and embraces diversity in all forms.
+- The project strives to be a meritocracy where more rights and responsibilities are granted to contributors who add value to the project.
+- The project has a public communication channel where community members can engage with maintainers.
+- The project adheres to, and enforces, the .NET Foundation Code of Conduct.
+- The project has a publicly available location where members can review documentation.
+
+It is expected that a project satisfies the majority of the criteria outlined above; however, it is also possible for a project to obtain a waiver for specific criteria. It is the responsibility of the Project Committee to collate the project information, perform an evaluation, and make recommendations on a project's eligibility to the Board of Directors. The Board of Directors has the right to approve or deny a project's eligibility. Projects will also be re-evaluated periodically to ensure they continue to satisfy the eligibility requirements.
+
+## Health Criteria
+
+Health criteria is designed to be measurable and is based on publicly accessible metrics which can be used to evaluate the health and maturity of a project on an ongoing basis. Metrics are based on values which can be readily obtained by anyone from GitHub and Nuget.
+
+- Age (based on the date on the LICENSE file in the repository or a date provided by the maintainer) 
+- Stars
+- Watchers
+- Forks
+- Used By
```

Comment:
Used By seems to only be an internal metric to other projects on GitHub.  Is this a fair criteria if the code isn't sourced on GitHub?

Reply:
To expand on that, are we restricting open-source projects to those hosted only on GitHub? Open source projects choosing to use an open source alternative (i.e. GitLab or similar) being locked out seems counterproductive.

Especially given that the Eligibility Criteria only *prefers* GitHub.

Reply:
The goal of this proposal is for the Project Committee to be able to collect health metrics on a monthly basis for projects. Github makes these metrics publicly available, so there can be full transparency. That being said, if a project is not using Github there is the provision that the health criteria can be collected from other sources or waived altogether so that no projects are excluded based on where  they are hosted.

Reply:
Will any of that information be documented? A GitLab-hosted project looking at this document would almost have to assume they'd be ineligible because they chose an open-source hosting platform, which is not a great impression.

Reply:
The Eligibility criteria could be modified to say:

-The project's code is easily discoverable and publicly accessible

And the Health criteria could be modified to say:

For convenience reasons, standard metrics are based on values which can be readily obtained by anyone from GitHub and Nuget; however, metrics from other open source hosting platforms will also be considered.